<?xml version="1.0" encoding="UTF-8"?>
<div xmlns="http://www.w3.org/1999/xhtml"><p/><b>&#214;deen </b> <i>et al</i> <b>. </b> <i>BMC Evolutionary Biology </i> <b>2011, </b> <b>11</b> <b>:313</b> <b>http://www.biomedcentral.com/1471-2148/11/313</b> <div><img src="../../../target/output/tree-1471-2148-11-313/image.g.1.1.png" width="20%"/><a href="../../../target/output/tree-1471-2148-11-313/image.g.1.1.svg" target="separate">SVG</a></div><div><img src="../../../target/output/tree-1471-2148-11-313/image.g.1.2.png" width="20%"/><a href="../../../target/output/tree-1471-2148-11-313/image.g.1.2.svg" target="separate">SVG</a></div><h1><div id="g.1.3"><p/><b>Evolution of ultraviolet vision in the largest avian</b> <b>radiation - the passerines</b> </div></h1>Anders &#214;deen <sup>1*</sup> , Olle H&#229;stad <sup>2,3 </sup> and Per Alstr&#246;m <sup>4</sup> <div><img src="../../../target/output/tree-1471-2148-11-313/image.g.1.5.png" width="20%"/><a href="../../../target/output/tree-1471-2148-11-313/image.g.1.5.svg" target="separate">SVG</a></div><p/><b>Background</b> Colour perception is one of the disciplines where birds excel. In general, interspecific variation in avian colour vision falls into two discrete classes, which are charac-terised by the spectral sensitivity of the most shortwave sensitive of the four single cones, the SWS1 [1] (the other, more long-wave sensitive being SWS2, MWS and LWS). The wavelength of maximum sensitivity ( &#955;  <sub>max</sub> ) of the SWS1 ranges either from 355-380 nm in the  <b>&#8216; </b> ultra-violet sensitive <b>&#8217; </b> class (UVS) or from 402-426 nm in the <p/><b>&#8216; </b> violet sensitive <b>&#8217; </b> (VS) (reviewed by [2] and [3]). The UVS class is optimised for ultraviolet sensitivity, but the VS class also has some degree of sensitivity in the ultravio-let spectrum (Figure 1). Changes in spectral sensitivity of the SWS1 are ecologically important, as spectral <div><img src="../../../target/output/tree-1471-2148-11-313/image.g.1.8.png" width="20%"/><a href="../../../target/output/tree-1471-2148-11-313/image.g.1.8.svg" target="separate">SVG</a></div>tuning affects the abilities of birds to accurately assess the quality of potential mates [4-14], spot elusive prey or detect other food items [15-20] and minimise visibi-lity of social signals in plumage coloration to predators [21]. Non-conservative substitutions in the SWS1 cone opsin protein that are located in the retinal binding pocket close enough to interact with the retinal chromo-phore may shift  &#955;  <sub>max </sub> from UVS to VS or vice versa [22,23]. Replacement of cysteine by serine in the 90 <sup>th</sup> amino acid (aa) position, the substitution Cys90Ser, or the reverse substitution, alone accounts for the whole shift (all aa residues named in this article are numbered according to bovine rhodopsin ([24]). Despite the simple nature of the mechanism, very few shifts have been described so far (e.g. review in [2]), suggesting that the SWS1 cone spectral sensitivity is under strong stabilising selection. The VS class is both ancestral and the most common to birds; its members are distributed <div><img src="../../../target/output/tree-1471-2148-11-313/image.g.1.10.png" width="20%"/><a href="../../../target/output/tree-1471-2148-11-313/image.g.1.10.svg" target="separate">SVG</a></div><p>see LICENSE</p><p/><b>&#214;deen </b> <i>et al</i> <b>. </b> <i>BMC Evolutionary Biology </i> <b>2011, </b> <b>11</b> <b>:313</b> <b>http://www.biomedcentral.com/1471-2148/11/313</b> <p/><b>Page 2 of 8</b> <div><img src="../../../target/output/tree-1471-2148-11-313/image.g.2.2.png" width="20%"/><a href="../../../target/output/tree-1471-2148-11-313/image.g.2.2.svg" target="separate">SVG</a></div>throughout the avian phylogeny [25,26]. Four or five independent shifts to UVS are currently known: in shorebirds, Passerida passerines, parrots, the rhea and presumably trogons [[26] and references therein, [27-30]]. The most promising group of birds for further investi-gation into the spectral tuning of SWS1 cone opsins is clearly the passerines (Passeriformes). This is the most species rich of all avian orders and the only one except the shorebirds (Charadriiformes) from which both classes of spectral sensitivity are known. Every investi-gated member of the Passerida clade of passerines belongs to the UVS class [26,27,31-38] and it has appeared that all other passerines can be placed in the VS category (see [26,27,37,39] and Browne,  <i>et al</i> . 2006 (GenBank only)). Recently, however, &#214;deen  <i>et al. </i> [40] argued that the evolution of ultraviolet sensitive vision in Passeriformes is more complex than a single VS to UVS opsin shift in an ancestor of Passerida, presenting molecular evidence for additional UV shifts from VS colour vision outside Passerida, in fairywrens, genus <i>Malurus</i> . Since spectral tuning of the cone is under the genetic control of a few amino acid residues in the opsin pro-tein, it is possible to quickly classify almost any bird as VS or UVS from a sample of genomic DNA [3,26]. The accuracy of this short-fragment genomic DNA approach in distinguishing UVS from VS species has been validated against all published MSP data [3]. We have employed this method to search for gross differ-ences in spectral tuning in a larger sample of passerine species than has been investigated before. Our aims with this survey were to assess how stable SWS1 cone sensitivity has been in the course of passerine evolu-tion and to trace the evolutionary sequence of shifts in spectral tuning. As the basis for the study, we have inferred a molecular phylogeny with sequence data from GenBank. <p/><b>Results</b> <p/><b>Opsin sequencing</b> We amplified the target fragment of the SWS1 opsin gene in 56 passerine species from 30 families and one falconiform species, Northern Crested Caracara  <i>Cara-</i><i>cara cheriway </i> (Additional file 1). Cycle sequencing pro-duced 50-107 bp long overlapping strands of identifiable nucleotides. Sequences have been deposited in the Eur-opean Nucleotide Archive (ENA) (accession numbers in Additional file 1). Amino acid translations spanning the spectral tuning sites 86, 90 and 93 [22] are presented in Additional file 1. For unknown reasons, we failed to amplify the SWS1 opsin in two species: Blue Jewel-bab-bler  <i>Ptilorrhoa caerulescens </i> (Psophodidae) and White-bellied Erpornis  <i>Erpornis zantholeuca </i> (Vireonidae). Position 90 held either serine or cysteine residues, sig-nifying VS and UVS opsins, respectively. Neither pheny-lalanine nor any other aa residues suggesting a potential major shift in spectral tuning through a non-conserva-tive substitution was present at site 86 (cf. [28]). How-ever, the Rifleman  <i>Acanthisitta chloris </i> holds leucine in spectral tuning site 93 (Leu93). This state is previously undescribed in birds and probably the result of a non-conservative substitution (see below). <p/><b>Phylogenetic reconstruction</b> The inferred tree (Figure 2) is mostly well resolved and, except for some short internal branches, well supported. It agrees well with previously published passerine trees based on fewer loci, such as a large-scale study of pas-serine relationships based on RAG-1 and -2 [41], as well as with more densely sampled studies of the Passerida based on ODC, myo and  &#946;  -fibrinogen intron 5 [42] and of the  <b>&#8216; </b> core Corvoidea <b>&#8217; </b> based on ODC, myo, RAG-1 and -2, and ND2 [43]. The character optimisation infers a minimum of six changes, two from VS to UVS, two from UVS to VS and two with uncertain direction due to three nodes with ambiguous ancestral states near the root (Figure 2). The number of inferred transitions are reduced to five, if clades  <i>b </i> and  <i>c </i> in Figure 2, which are reconstructed with low support, are collapsed, and  <i>Cnemophilus </i> is placed as sister to the  <b>&#8216; </b> core Corvoidea <b>&#8217; </b> clade (Additional file 2). On the contrary, if all  <b>&#8216; </b> basal <b>&#8217; </b> nodes with posterior probability &lt; 0.95 are collapsed, eight transitions are inferred (Additional file 3). The ancestor to all passer-ines is reconstructed as UVS when only parrots (UVS) form the outgroup, whereas it is ambiguous when falco-niforms (VS) are also included in the outgroup (as shown in Figure 2). <p/><b>&#214;deen </b> <i>et al</i> <b>. </b> <i>BMC Evolutionary Biology </i> <b>2011, </b> <b>11</b> <b>:313</b> <b>http://www.biomedcentral.com/1471-2148/11/313</b> <p/><b>Page 3 of 8</b> <div><img src="../../../target/output/tree-1471-2148-11-313/image.g.3.2.png" width="20%"/><a href="../../../target/output/tree-1471-2148-11-313/image.g.3.2.svg" target="separate">SVG</a></div><p/><b>&#214;deen </b> <i>et al</i> <b>. </b> <i>BMC Evolutionary Biology </i> <b>2011, </b> <b>11</b> <b>:313</b> <b>http://www.biomedcentral.com/1471-2148/11/313</b> <p/><b>Page 4 of 8</b> <p/><b>Discussion</b> The evolution of colour vision in birds has apparently involved many more than the four to five independent shifts between SWS1 cone opsin types that previously available data have indicated. In Passeriformes, the pic-ture is indeed more complex than a single VS to UVS opsin shift in an ancestor of Passerida; our phylogenetic trait mapping reveals more variation among the passer-ines than what is known from other birds combined. The class of colour vision of the most recent common ancestor to the passerines is ambiguous as a result of different states in parrots (UVS) and falconiforms (VS). As parrots are suggested to be the sister group to the passerines [44], the passerine ancestor is hypothesised to be UVS (as inferred when falconiforms are excluded from the optimisation analyses). In the subsequent pas-serine radiation, colour vision has changed between UVS and VS at least eight times, including two shifts in the family Maluridae (signified by # in Figure 2) [40]. It is interesting to note that two major clades, the Pas-serida/Petroicidae and the suboscines, are homogeneous with respect to colour vision (albeit different classes), whereas there is considerably more variation in the rest of the tree, particularly within Maluridae. It is not imme-diately apparent what the selective advantage of either VS or UVS might be, as both classes are present in taxa represented on all continents and in similar habitats. For example, the ancestor to the mostly forest-dwelling New World Vireonidae has changed from VS to UVS, whereas the mainly forest-inhabiting New World suboscines are VS. Moreover, the UVS Passerida/Petroicidae clade is mainly distributed in Afro-Eurasia and the New World, with few representatives in Australasia; the VS suboscines occur predominantly in the New World, with relatively few species in Africa, Madagascar, Asia and Australasia; the mostly VS, but also UVS,  <b>&#8216; </b> basalmost <b>&#8217; </b> oscines ( <i>Menura</i> -<i>Orthonyx</i> ) plus  <i>Philesturnus</i> ,  <i>Cnemophilus </i> and <i>Toxorhampus </i> are chiefly Australasian; and the  <b>&#8216; </b> core Cor-voidea <b>&#8217; </b> have representatives on all continents. The ances-tral area of the passerines has been inferred to be eastern Gondwana,  <i>i.e</i> . proto-Australasia [45,46], whereas the <i>Philesturnus</i> / <i>Cnemophilus</i> / <i>Toxorhampus</i> / <b>&#8217; </b> core Corvoi-dea <b>&#8217; </b> clade has been inferred to have originated in the proto-Papuan archipelago [43]. Unlike Maluridae (represented by  <i>Amytornis </i> in Figure 2), none of the multi-sampled families from the present study are polytypic in gross tuning of the SWS1 opsin, but either contain VS or UVS taxa. In common with the Australian-Papuan Maluridae, the African-Eurasian genus  <i>Motacilla </i> (wagtails) is rich in species and subspe-cies with strong differentiation in male plumage colora-tion. Yet, all the members of  <i>Motacilla </i> sampled have base-pair identical SWS1 opsin gene fragments. Unknown differences in other traits than SWS1 could explain why UVS-VS shifts are so unequally distributed between Passerida/Petroicidae and other passerines. Spectral tuning of the SWS1 cone opsin seems to be under strong stabilising selection in birds [30], likely partly due to gross spectral tuning shifts carrying costs, such as increased retinal photooxidation and ocular light scattering (reviewed in [47]). In addition, acute short wave colour vision is dependent on a complex of co-adapted physiological traits. Ultraviolet absorption by the ocular media is usually stronger in VS than in UVS species (reviewed in [48,49]), and the SWS2 cone class (sensitive to  <b>&#8216; </b> blue <b>&#8217; </b> ,  &#955;  <sub>max </sub> 451-480 nm) is shifted toward shorter wavelengths in UVS than in VS birds (see review in [2]). A shortwave shift in  &#955;  <sub>max </sub> of a VS SWS1 pigment will produce a virtually negligible increase in UV sensi-tivity, unless it is accompanied by an increased UV transmission in the ocular media [48]. A shortwave shifted SWS1 will cause an uneven distribution of sensi-tivities across the spectrum, possibly deteriorating col-our discriminability, if not followed by a shift of the SWS2 pigment towards the range vacated by SWS1. A plausible evolutionary scenario is that increased UV transmittance in an ancestor of passerines relaxed stabi-lising selection on SWS1 spectral tuning, allowing a VS to UVS shift of that opsin to reach fixation (see [48]) (the only close relative to passerines studied in this respect is one with a UV transparent ocular media: Bud-gerigar  <i>Melopsittacus undulatus </i> [49]). Later, a short-wave shift in SWS2 spectral tuning in an ancestor of Passerida/Petroicidae may have improved spectral discri-minability of UVS retinae and thereby reinstated stabi-lising selection on SWS1 spectral tuning. In agreement with this scenario, Passerida species where this cone has been examined by MSP mostly show shortwave shifted SWS2  &#955;  <sub>max </sub> compared to VS species from other avian orders (see review [2]), but unfortunately the character state is not known from any VS or non-Passerida pas-serine. The realised spectral sensitivities of the two shortwave sensitive cones in non-Passerida passerines may be different from that of characteristic VS and UVS types (cf. review in [2]). Due to a general paucity of data on SWS2  &#955;  <sub>max </sub> and ocular transmission, one can merely speculate at this point. Shifts from UVS to VS in passerines seem to be con-trolled exclusively by the Cys90Ser substitution. Car-valho  <i>et al. </i> [28] has reported a significant short wavelength tuning effect of substitution Ser86Phe, but we did not find phenylalanine in position 86 (Phe86) in any of the species we sequenced. The effects of all tun-ing site substitutions found in this study are known [22,23], with one exception: a leucine residue in the spectral tuning site 93 of the Rifleman  <i>Acanthisitta</i> <p/><b>&#214;deen </b> <i>et al</i> <b>. </b> <i>BMC Evolutionary Biology </i> <b>2011, </b> <b>11</b> <b>:313</b> <b>http://www.biomedcentral.com/1471-2148/11/313</b> <p/><b>Page 5 of 8</b> <i>chloris</i> . The most likely evolutionary event given the phylogeny in Figure 2 is the substitution Thr93Leu. This is a non-conservative substitution, changing from polar and mildly hydrophilic threonine to nonpolar, hydro-phobic leucine. Site 93 is located in the retinal binding pocket, on the inner side of the opsin <b>&#8217; </b> s alpha-helices close enough to interact with the retinal chromophore. Thr93Leu is therefore potentially important to spectral tuning (see [50]), but its effect in the avian SWS1 opsin is presently unknown. The reconstruction of the evolution of colour vision in passerines is conditional on the true phylogeny having been inferred. The tree inferred here is based on a larger number of loci than any previous study of passerine relationships. As it is overall well supported by the data, and in good agreement with previous analyses of differ-ent datasets (e.g. [41-43], it is a well founded hypothesis of relationships. Although the tree has some nodes with low posterior probability, only two of these affect the interpretation of the evolution of colour vision (indi-cated by  <i>a </i> and  <i>b </i> in Figure 2). With respect to the poorly supported node  <i>a</i> , switching position between <i>Pomatostomus </i> and  <i>Orthonyx</i> , as is possible if this node is collapsed, would require one more step in the VS/ UVS optimisation and result in eight further internal nodes having ambiguous states. Further collapsing of poorly supported  <b>&#8216; </b> basal <b>&#8217; </b> nodes leads to more steps in the UV optimisation. Accordingly, the opsin data lend further support to the present topology, and the favoured topology results in a more parsimonious opti-misation than the alternative topologies. However, if the insignificantly supported nodes  <i>b </i> and  <i>c </i> are collapsed, and  <i>Cnemophilus </i> is placed as sister to the  <b>&#8216; </b> core Corvoi-dea <b>&#8217; </b> , one fewer change is required to explain the evolu-tion of colour vision. More sequence data might resolve this issue in the future. <p/><b>Conclusions</b> Spectral tuning of the SWS1 cone opsin in birds is a trait of great ecological importance. It appears to be under strong stabilising selection and varies categori-cally between different clades in the avian phylogenetic tree. In the radiation of the very species rich order Passeriformes, sequencing of a small gene fragment allows us to map and trace the change from one stable state to the other. Clades with variation in the colour vision system are nested among clades with a see-mingly stable VS or UVS state, providing a rare oppor-tunity to understand how an ecologically important trait under simple genetic control may co-evolve with, and be stabilised by, associated traits in a character complex. <p/><b>Methods</b> <p/><b>Opsin sequencing</b> We isolated genomic DNA from tissues and blood sam-ples, which we took from live birds in the field or bor-rowed from museums and colleagues. As far as feasible we included representatives from clades of expected importance to tracing the radiation of passerines. As demonstrated in the fairywren case [40], closely related groups of passerine taxa might be polytypic with respect to spectral tuning of the SWS1 opsin. Acknowledging this possibility, we sampled multiple taxa in selected genera. We used a GeneMole <sup>&#174; </sup> automated nucleic acid extraction instrument (Mole Genetics) and the DNeasy Blood and Tissue Kit (QIAGEN) for the DNA isolation. Standard procedures were applied. With the same pro-tocol and primers as are described in [3] and [26] we then amplified a fragment of the SWS1 opsin gene, con-taining the aa residues at positions 81-94, all located in the 2 <sup>nd </sup> &#945;  -helical transmembrane region. We translated the DNA sequences into amino acids to identify the spectral tuning sites 86, 90, and 93 of SWS1 [22,23]. Then we calculated  &#955;  <sub>max </sub> from the tuning sites following  <i>in vitro </i> changes in  &#955;  <sub>max </sub> reported by Wilkie  <i>et</i> <i>al. </i> [22]. We assumed the effects of the key tuning sites to be additive. This assumption should provide reason-able approximations of  &#955;  <sub>max </sub> [51], although it disregards potential interactions between the tuning sites (see [52]). <p/><b>Phylogenetic reconstruction</b> A phylogeny was inferred from sequences in GenBank (Additional file 4). We made a search in GenBank for all falconiform, psittaciform and passeriform species for which we and others had sequenced the SWS1 opsin gene. We then selected eight loci for which sequences were available for a large proportion of these species or members of the same genus: the mitochondrial cyto-chrome  <i>b </i> (cyt <i>b</i> ) and NADH dehydrogenase II (ND2) genes, the nuclear myoglobin gene, intron 2 (myo), ornithine decarboxylase gene, introns 6 to 7 (ODC), transforming growth factor beta-2 gene, intron 5 (TGF &#946;  2), and the protein-coding nuclear c-myc exon 3 and recombination-activating protein 1 and 2 genes (RAG-1, RAG-2). Sequences were aligned using the MUSCLE web server http://www.drive5.com/muscle; some manual adjustment was necessary for the non-coding sequences. The alignment is in Additional file 5. The phylogeny was estimated by Bayesian inference using MrBayes 3.1.2 [53,54], with the sequences conca-tenated and partitioned by locus, using rate multipliers to allow different rates for the different partitions [55,56]. Ambiguous base pairs and indels were treated as missing data. As outgroups, we chose two <p/><b>&#214;deen </b> <i>et al</i> <b>. </b> <i>BMC Evolutionary Biology </i> <b>2011, </b> <b>11</b> <b>:313</b> <b>http://www.biomedcentral.com/1471-2148/11/313</b> <p/><b>Page 6 of 8</b> falconiforms and two parrots, as these have been shown to be closely related to passerines [44,57]. Appropriate substitution models were determined based on the Baye-sian Information Criterion [58] calculated by jModelTest version 0.1.1 [59]. For cyt <i>b</i> , ND2 and RAG-1, the best-fit model was the general time-reversible (GTR) model [60-62], assuming rate variation across sites according to a discrete gamma distribution with four rate cate-gories ( &#915;  ; [63]) and an estimated proportion of invariant sites (I; [64]) (GTR+ &#915;  +I). For the other loci, the Hase-gawa-Kishino-Yano (HKY) model [65] plus  &#915;  was selected (HKY+ &#915;  ), with the addition of I for myoglobin and c-myc (HKY+ &#915;  +I). Posterior probabilities (PPs) were calculated in MrBayes using default priors. Four Metropolis-coupled MCMC chains with incremental heating temperature 0.1 were run for 7 &#215; 10 <sup>6 </sup> genera-tions and sampled every 1000 generations. Chain likeli-hood and other parameter values and effective sample sizes (&gt; 200) were inspected in Tracer 1.5.0 [66]. The first 25% of the generations were discarded as  <b>&#8216; </b> burn-in <b>&#8217; </b> , well after stationarity of most chain likelihood values had been established, and the posterior probability was estimated for the remaining generations. The analysis was run eight times, and the topologies and posterior probabilities were compared by eye and by the mean estimates and the corresponding standard errors. Optimisation of the VS/UVS character was performed by parsimony in MacClade 4.08 [67], using default set-tings (polytomies treated as soft). <p/><b>List of abbreviations</b> &#955;  <sub>max</sub> : wavelength of maximum absorbance; MSP: microspectrophotometry; SWS1: short-wavelength sensitive pigment, type one; VS: violet sensitive; UVS: ultraviolet sensitive. <p>see ACKNOWLEDGMENT</p>Tissue samples used for this study were kindly put at our disposal by The Australian Museum, Sydney, Nicklas Backstr&#246;m and Sofia Berlin at the Department of Evolutionary Biology, Allan Baker and Oliver Haddrath at the Royal Ontario Museum, Toronto, Peter Halvarsson at the Department of Population Biology, and Christer Hemborg, Damian Dowling and Anna Qvarnstr&#246;m at the Department of Animal Ecology, Uppsala University, Przemek Chylarecki, The Field Museum of Natural History, Chicago, Fyris Zoo, Uppsala, Louisiana State University Museum of Natural Science, Baton Rouge, the Swedish Museum of Natural History, Stockholm, J&#252;rgen Tomiuk, University of Washington Burke Museum, Seattle and the Zoological Museum of the University of Copenhagen, Copenhagen. Christer Hemborg, Anders Lindstr&#246;m, Marta Vila Taboada, Teet Sirotkin, Staffan Ulfstrand and Malcom Wilson collected samples in the field together with A&#214;. This work was funded by the Swedish Research Council Formas, the Royal Swedish Academy of Sciences, Stiftelsen f&#246;r zoologisk forskning and Carl Tryggers Stiftelse f&#246;r Vetenskaplig Forskning. <p>see AUTHORS</p>1 Department of Animal Ecology, Uppsala University, Norbyv&#228;gen 18D, S-752 36 Uppsala, Sweden.  <sup>2</sup> Department of Evolutionary Organismal Biology, Uppsala University, Norbyv&#228;gen 18A, S-752 36 Uppsala, Sweden. 3 Department of Anatomy, Physiology and Biochemistry, Swedish University of Agricultural Sciences, P.O. Box 7011, S-750 07, Uppsala, Sweden.  <sup>4</sup> Swedish Species Information Centre, Swedish University of Agricultural Sciences, Box 7007, S-750 07 Uppsala, Sweden. <p/><b>Authors</b> <b>&#8217; </b> <b>contributions</b> A&#214; compiled the tissue material and carried out the genetic studies. A&#214; and OH conceived of the study, interpreted the opsin results and compiled most of the molecular marker data. PA compiled some of the molecular marker data, performed sequence alignment and reconstructed the phylogeny. All authors participated in the trait state mapping, drafted the manuscript, and read and approved it. <p/><b>Additional material</b> <div><img src="../../../target/output/tree-1471-2148-11-313/image.g.6.12.png" width="20%"/><a href="../../../target/output/tree-1471-2148-11-313/image.g.6.12.svg" target="separate">SVG</a></div><p/><b>Received: 24 March 2011 Accepted: 24 October 2011</b> <b>Published: 24 October 2011</b> <p/><b>References</b> <ul><li>1. Cuthill IC, Partridge JC, Bennett ATD, Church SC, Hart NS, Hunt S: <p/><b>Ultraviolet vision in birds. </b> <i>Adv Stud Behav </i> 2000,  <b>29</b> :159-214. </li><li>2. Hart NS, Hunt DM:  <b>Avian visual pigments: characteristics, spectral tuning</b> <b>and evolution. </b> <i>Am Nat </i> 2007,  <b>169(Supplement)</b> :S7-S26. </li><li>3. &#214;deen A, Hart NS, H&#229;stad O:  <b>Assessing the use of genomic DNA as a</b> <b>predictor of the maximum absorbance wavelength of avian SWS1 opsin</b> <b>visual pigments. </b> <i>J Comp Physiol A </i> 2009,  <b>195</b> :167-173. </li><li>4. Maier EJ:  <b>To deal with </b> <b>&#8216; &#8217; </b> <b>invisible</b> <b>&#8217; &#8217; </b> <b>: on the biological importance of</b> <b>ultraviolet sensitivity in birds. </b> <i>Naturwissenschaften </i> 1993,  <b>80</b> :476-478. </li><li>5. Bennett ATD, Cuthill IC, Partridge JC, Maier EJ:  <b>Ultraviolet vision and mate</b> <b>choice in zebra finches. </b> <i>Nature </i> 1996,  <b>380</b> :433-435. </li><li>6. Bennett ATD, Cuthill IC, Partridge JC, Lunau K:  <b>Ultraviolet plumage colors</b> <b>predict mate preferences in starlings. </b> <i>Proc Natl Acad Sci USA </i> 1997, <p/><b>94</b> :8618-8621. </li><li>7. Andersson S, Amundsen T:  <b>Ultraviolet colour vision and ornamentation in</b> <b>bluethroats. </b> <i>Proc R Soc B </i> 1997,  <b>264</b> :1587-1591. </li><li>8. Hunt S, Cuthill IC, Swaddle JP, Bennett ATD:  <b>Ultraviolet vision and band</b> <b>colour preferences in female zebra finches, </b> <i>Taeniopygia guttata</i> <b>. </b> <i>Anim</i> <i>Behav </i> 1997,  <b>54</b> :1382-1392. </li><li>9. Hunt S, Bennett ATD, Cuthill IC, Griffiths R:  <b>Blue tits are ultraviolet tits. </b> <i>Proc</i> <i>R Soc B </i> 1998,  <b>265</b> :451-455. </li><li>10. Hunt S, Cuthill IC, Bennett ATD, Griffiths R:  <b>Preferences for ultraviolet</b> <b>partners in the blue tit. </b> <i>Anim Behav </i> 1999,  <b>58</b> :809-815. </li><li>11. Johnsen A, Andersson S, &#214;rnborg J, Lifjeld JT:  <b>Ultraviolet plumage</b> <b>ornamentation affects social mate choice and sperm competition in</b> </li></ul><p/><b>&#214;deen </b> <i>et al</i> <b>. </b> <i>BMC Evolutionary Biology </i> <b>2011, </b> <b>11</b> <b>:313</b> <b>http://www.biomedcentral.com/1471-2148/11/313</b> <p/><b>Page 7 of 8</b> <ul><li>12. Andersson S, &#214;rnborg J, Andersson M:  <b>Ultraviolet sexual dimorphism and</b> <b>assortative mating in blue tits. </b> <i>Proc R Soc B </i> 1998,  <b>265</b> :445-450. </li><li>13. Siitari H, Honkavaara J, Huhta E, Viitala J:  <b>Ultraviolet reflection and female</b> <b>mate choice in the pied flycatcher, </b> <i>Ficedula hypoleuca</i> <b>. </b> <i>Anim Behav </i> 2002, <p/><b>63</b> :97-102. </li><li>14. Gaibani G, Csermely D, Zampiga E:  <b>Ultraviolet reflectance and female</b> <b>mating preferences in the common kestrel (</b> <i>Falco tinnunculus</i> <b>). </b> <i>Canadian</i> <i>J Zool </i> 2008,  <b>86</b> :479-483, (5). </li><li>15. Burkhardt D:  <b>Birds, berries and UV. </b> <i>Naturwissenschaften </i> 1982,  <b>69</b> :153-157. </li><li>16. Viitala J, Korpim&#228;ki E, Palokangas P, Koivula M:  <b>Attraction of kestrels to</b> <b>vole scent marks visible in ultraviolet light. </b> <i>Nature </i> 1995,  <b>373</b> :425-427. </li><li>17. Church SC, Bennett ATD, Cuthill IC, Partridge JC:  <b>Ultraviolet cues affect the</b> <b>foraging behaviour of blue tits. </b> <i>Proc R Soc B </i> 1998,  <b>265</b> :1509-1514. </li><li>18. Koivula M, Viitala J:  <b>Rough-legged buzzards use vole scent marks to</b> <b>assess hunting areas. </b> <i>Avian Biol </i> 1999,  <b>30</b> :329-330. </li><li>19. Siitari H, Honkavaara J, Viitala J:  <b>Ultraviolet reflection of berries attracts</b> <b>foraging birds. A laboratory study with redwings (</b> <i>Turdus iliacus</i> <b>) and</b> <b>bilberries (</b> <i>Vaccinium myrtillus</i> <b>). </b> <i>Proc R Soc B </i> 1999,  <b>266</b> :2125-2129. </li><li>20. Probst R, Pavlicev M, Viitala J:  <b>UV reflecting vole scent marks attract a</b> <b>passerine: the great grey shrike </b> <i>Lanius excubitor</i> <b>. </b> <i>J Avian Biol </i> 2002, <p/><b>33</b> :437-440. </li><li>21. H&#229;stad O, Victorsson J, &#214;deen A:  <b>Differences in color vision make</b> <b>passerines less conspicuous in the eyes of their predators. </b> <i>Proc Natl Acad</i> <i>Sci USA </i> 2005,  <b>102</b> :6391-6394. </li><li>22. Wilkie SE, Robinson PR, Cronin TW, Poopalasundaram S, Bowmaker JK, Hunt DM:  <b>Spectral tuning of avian violet- and ultraviolet-sensitive visual</b> <b>pigments. </b> <i>Biochemistry </i> 2000,  <b>39</b> :7895-7901. </li><li>23. Yokoyama S, Radlwimmer FB, Blow NS:  <b>Ultraviolet pigments in birds</b> <b>evolved from violet pigments by a single amino acid change. </b> <i>Proc Natl</i> <i>Acad Sci USA </i> 2000,  <b>97</b> :7366-7371. </li><li>24. Nathans J, Hogness DS:  <b>Isolation, sequence analysis, and intron-exon</b> <b>arrangement of the gene encoding bovine rhodopsin. </b> <i>Cell </i> 1983, <p/><b>34</b> :807-814. </li><li>25. Yokoyama S:  <b>Molecular evolution of color vision in vertebrates. </b> <i>Gene</i> 2002,  <b>300</b> :69-78. </li><li>26. &#214;deen A, H&#229;stad O:  <b>Complex distribution of avian color vision systems</b> <b>revealed by sequencing the SWS1 opsin from total DNA. </b> <i>Mol Biol Evol</i> 2003,  <b>20</b> :855-861. </li><li>27. &#214;deen A, H&#229;stad O:  <b>Pollinating birds differ in spectral sensitivity. </b> <i>J Comp</i> <i>Physiol A </i> 2010,  <b>196</b> :91-96. </li><li>28. Carvalho LS, Cowing JA, Wilkie SE, Bowmaker JK, Hunt DM:  <b>The molecular</b> <b>evolution of avian ultraviolet- and violet-sensitive visual pigments. </b> <i>Mol</i> <i>Biol Evol </i> 2007,  <b>24</b> :1843-1852. </li><li>29. Carvalho LS, Knott B, Berg ML, Bennett ATD, Hunt DM:  <b>Ultraviolet-sensitive</b> <b>vision in long-lived birds. </b> <i>Proc R Soc B </i> 2010, published online before print July 28. </li><li>30. &#214;deen A, H&#229;stad O, Alstr&#246;m P:  <b>Evolution of ultraviolet vision in</b> <b>shorebirds (Charadriiformes). </b> <i>Biol Lett </i> 2010,  <b>6</b> :370-374. </li><li>31. Maier EJ, Bowmaker JK:  <b>Colour vision in the passeriform bird, </b> <i>Leothrix</i> <i>lutea</i> <b>: correlation of visual pigment absorbance and oil droplet</b> <b>transmission with spectral sensitivity. </b> <i>J Comp Physiol A </i> 1993,  <b>172</b> :295-301. </li><li>32. Bowmaker JK, Heath LA, Wilkie SE, Hunt DM:  <b>Visual pigments and oil</b> <b>droplets from six classes of photoreceptor in the retinas of birds. </b> <i>Vision</i> <i>Res </i> 1997,  <b>37</b> :2183-2194. </li><li>33. Hart NS, Partridge JC, Cuthill IC:  <b>Visual pigments, oil droplets and cone</b> <b>photoreceptor distribution in the European starling (</b> <i>Sturnus vulgaris</i> <b>). </b> <i>J</i> <i>Exp Biol </i> 1998,  <b>201</b> :1433-1446. </li><li>34. Das D, Wilkie SE, Hunt DM, Bowmaker JK:  <b>Visual pigments and oil droplets</b> <b>in the retina of a passerine bird, the canary </b> <i>Serinus canaria</i> <b>:</b> <b>microspectrophotometry and opsin sequences. </b> <i>Vision Res </i> 1999, <p/><b>39</b> :2801-2815. </li><li>35. Hart NS, Partridge JC, Bennett ATD, Cuthill IC:  <b>Visual pigments, cone oil</b> <b>droplets and ocular media in four species of estrildid finch. </b> <i>J Comp</i> <i>Physiol A </i> 2000,  <b>186</b> :681-694. </li><li>36. Hart NS, Partridge JC, Cuthill IC, Bennett ATD:  <b>Visual pigments, oil droplets,</b> <b>ocular media and cone photoreceptor distribution in two species of</b> <b>passerine bird: the blue tit (</b> <i>Parus caeruleus </i> <b>L.) and the blackbird (</b> <i>Turdus</i> <i>merula </i> <b>L.). </b> <i>J Comp Physiol A </i> 2000,  <b>186</b> :375-387. </li></ul><ul><li>37. &#214;deen A, H&#229;stad O:  <b>New primers for the avian SWS1 pigment opsin</b> <b>gene reveal new amino acid configurations in spectral sensitivity tuning</b> <b>sites. </b> <i>J Hered </i> 2009,  <b>100</b> :784-789. </li><li>38. Beason RC, Loew ER:  <b>Visual pigment and oil droplet characteristics of the</b> <b>bobolink (</b> <i>Dolichonyx oryzivorus</i> <b>), a new world migratory bird. </b> <i>Vision Res</i> 2008,  <b>48</b> :1-8. </li><li>39. Seddon N, Tobias JA, Eaton M, &#214;deen A:  <b>Human vision can provide a</b> <b>valid proxy for avian perception of sexual dichromatism. </b> <i>Auk </i> 2010, <p/><b>127</b> :283-292. </li><li>40. &#214;deen A, Pruett-Jones S, Driskell AC, Armenta JK, H&#229;stad O:  <b>Multiple shifts</b> <b>between violet and ultraviolet vision in a family of passerine birds with</b> <b>associated changes in plumage coloration. </b> <i>Proc R Soc B </i> 2011, published online before print October 5. </li><li>41. Barker FK, Cibois A, Schikler P, Feinstein J, Cracraft J:  <b>Phylogeny and</b> <b>diversification of the largest avian radiation. </b> <i>Proc Natl Acad Sci USA </i> 2004, <p/><b>101</b> :11040-11045. </li><li>42. Johansson U, Fjelds&#229; J, Bowie RCK:  <b>Phylogenetic relationships within</b> <b>Passerida (Aves: Passeriformes): a review and a new molecular</b> <b>phylogeny based on three nuclear intron markers. </b> <i>Mol Phylogenet Evol</i> 2008,  <b>48</b> :858-876. </li><li>43. J&#248;nsson KA, Fabre P-H, Ricklefs RE, Fjelds&#229; J:  <b>Major global radiation of</b> <b>corvoid birds originated in the proto-Papuan archipelago. </b> <i>Proc Natl Acad</i> <i>Sci USA </i> 2011,  <b>108</b> :2328-2333. </li><li>44. Hackett SJ, Kimball RT, Reddy S, Bowie RCK, Braun EL, Braun MJ, Chojnowski JL, Cox WA, Han K-L, Harshman J, Huddleston CJ, Marks BD, Miglia KJ, Moore WS, Sheldon FH, Steadman DW, Witt CC, Yuri T:  <b>A</b> <b>phylogenomic study of birds reveals their evolutionary history. </b> <i>Science</i> 2008,  <b>320</b> :1763-1768. </li><li>45. Barker FK, Barrowclough GF, Groth JG:  <b>A phylogenetic hypothesis for</b> <b>passerine birds: taxonomic and biogeographic implications of an</b> <b>analysis of nuclear DNA sequence data. </b> <i>Proc R Soc B </i> 2002,  <b>269</b> :295-308. </li><li>46. Ericson PGP, Christidis L, Cooper A, Irestedt M, Jackson J, Johansson US, Norman JA:  <b>A Gondwanan origin of passerine birds supported by DNA</b> <b>sequences of the endemic New Zealand wrens. </b> <i>Proc R Soc B </i> 2002, <p/><b>269</b> :235-241. </li><li>47. Bennett ATD, Cuthill IC:  <b>Ultraviolet vision in birds: what is its function?</b> <i>Vision Res </i> 1994,  <b>34</b> :1471-1478. </li><li>48. H&#229;stad O, Partridge JC, &#214;deen A:  <b>Ultraviolet photopigment sensitivity and</b> <b>ocular media transmittance in gulls, with an evolutionary perspective. </b> <i>J</i> <i>Comp Physiol A </i> 2009,  <b>195</b> :585-590. </li><li>49. Lind O, Kelber A:  <b>Avian colour vison: Effects of variation in receptor</b> <b>sensitivity and noise data on model predictions as compared to</b> <b>behavioural results. </b> <i>Vision Res </i> 2009,  <b>49</b> :1939-1947. </li><li>50. Wilkie SE, Vissers PMAM, Das D, DeGrip WJ, Bowmaker JK, Hunt DM:  <b>The</b> <b>molecular basis for UV vision in birds: spectral characteristics, cDNA</b> <b>sequence and retinal localization of the UV-sensitive visual pigment of</b> <b>the budgerigar (</b> <i>Melopsittacus undulatus</i> <b>). </b> <i>Biochem J </i> 1998,  <b>330</b> :541-547. </li><li>51. Yokoyama S, Tada T:  <b>The spectral tuning in the short wavelength-</b><b>sensitive type 2 pigments. </b> <i>Gene </i> 2003,  <b>306</b> :91-98. </li><li>52. Shi Y, Radlwimmer FB, Yokoyama S:  <b>Molecular genetics and the evolution</b> <b>of ultraviolet vision in vertebrates. </b> <i>Proc Natl Acad Sci USA </i> 2001, <p/><b>98</b> :11731-6. </li><li>53. Huelsenbeck JP, Ronquist F:  <b>MRBAYES: Bayesian inference of phylogeny.</b> <i>Bioinformatics </i> 2001,  <b>17</b> :754-755. </li><li>54. Huelsenbeck JP, Ronquist F:  <b>MRBAYES version 3.1.2. </b> 2005 [http://mrbayes. scs.fsu.edu/download.php]. </li><li>55. Nylander JAA, Ronquist F, Huelsenbeck JP, Nieves-Aldrey JL:  <b>Bayesian</b> <b>phylogenetic analysis of combined data. </b> <i>Syst Biol </i> 2004,  <b>53</b> :47-67. </li><li>56. Ronquist F, Huelsenbeck JP:  <b>MRBAYES 3: Bayesian phylogenetic inference</b> <b>under mixed models. </b> <i>Bioinformatics </i> 2003,  <b>19</b> :1572-1574. </li><li>57. Ericson PGP, Anderson CL, Britton T, Elzanowski A, Johansson US, K&#228;llersj&#246; M, Ohlson JI, Parsons TJ, Zuccon D, Mayr G:  <b>Diversification of</b> <b>Neoaves: integration of molecular sequence data and fossils. </b> <i>Biol Lett</i> 2006,  <b>2</b> :543-547. </li><li>58. Schwarz G:  <b>Estimating the dimension of a model. </b> <i>Ann Stat </i> 1978, <p/><b>6</b> :461-464. </li><li>59. Posada D:  <b>jModelTest: Phylogenetic Model Averaging. </b> <i>Mol Biol Evol </i> 2008, <p/><b>25</b> :1253-1256. </li><li>60. Lanave C, Preparata C, Saccone C, Serio G:  <b>A new method for calculating</b> <b>evolutionary substitution rates. </b> <i>J Mol Evol </i> 1984,  <b>20</b> :86-93. </li></ul><p/><b>&#214;deen </b> <i>et al</i> <b>. </b> <i>BMC Evolutionary Biology </i> <b>2011, </b> <b>11</b> <b>:313</b> <b>http://www.biomedcentral.com/1471-2148/11/313</b> <p/><b>Page 8 of 8</b> <ul><li>61. Tavar&#233; S:  <b>Some probabilistic and statistical problems on the analysis of</b> <b>DNA sequences. </b> <i>Lect Mater Life Sci </i> 1986,  <b>17</b> :57-86. </li><li>62. Rodr&#237;guez J, Oliver L, Mar&#237;n A, Medina R:  <b>The general stochastic model of</b> <b>nucleotide substitution. </b> <i>J Theor Biol </i> 1990,  <b>142</b> :485-501. </li><li>63. Yang Z:  <b>Maximum likelihood phylogenetic estimation from DNA</b> <b>sequences with variable rates over sites: approximate methods. </b> <i>J Mol</i> <i>Evol </i> 1994,  <b>39</b> :306-314. </li><li>64. Gu X, Fu Y-X, Li W-H:  <b>Maximum likelihood estimation of the</b> <b>heterogeneity of substitution rate among nucleotide sites. </b> <i>Mol Biol Evol</i> 1995,  <b>12</b> :546-557. </li><li>65. Hasegawa M, Kishino H, Yano T:  <b>Dating of the human-ape splitting by a</b> <b>molecular clock of mitochondrial DNA. </b> <i>J Mol Evol </i> 1985,  <b>22</b> :160-174. </li><li>66. Rambaut A, Drummond AJ:  <b>Tracer version 1.5.0. </b> 2009 [http://beast.bio.ed. ac.uk]. </li><li>67. Maddison WP, Maddison DR:  <b>MacClade: Analysis of phylogeny and</b> <b>character evolution. Version 4. </b> Sunderland, Massachusetts, Sinauer Associates; 2005. </li><li>68. Hart NS:  <b>Vision in the peafowl (Aves: </b> <i>Pavo cristatus</i> <b>). </b> <i>J Exp Biol </i> 2002, <p/><b>205</b> :3925-3935. </li><li>69. Gill F, Donsker D, (eds):  <b>IOC World Bird Names (version 2.9). </b> 2011 [http:// www.worldbirdnames.org/], Accessed Aug. 23 2011. </li></ul><div><img src="../../../target/output/tree-1471-2148-11-313/image.g.8.3.png" width="20%"/><a href="../../../target/output/tree-1471-2148-11-313/image.g.8.3.svg" target="separate">SVG</a></div><div><img src="../../../target/output/tree-1471-2148-11-313/image.g.8.4.png" width="20%"/><a href="../../../target/output/tree-1471-2148-11-313/image.g.8.4.svg" target="separate">SVG</a></div></div>
